src/styles/dashboard/notice.scss

```diff
@@ -8,9 +8,9 @@
 	}
 
 	.wp-notification {
```

Comment:
should this classname be `wp-notifications`?

Reply:
no because it refer to a single notification... maybe better to rename into .notification at this point?

Reply:
The right way in this whole prefix scheme would be:

```css
.wp-notifications__notification
```

Its not quite BEM, because we are throwing a namespace in there too. it makes sense to me it is a notification element in the `wp-notifications` css namespace.

Reply:
I looking into how `@wordpress/components` are namespaced and they are just under `.components` maybe we can just use `.notifications`?

Reply:
HMMM 🤔 looks like a little rough to use something so generic! 

Why not jail all the plugin code with `#id + .className`? This way we have a style that always takes priority while remaining fairly compact and readable (ps I hate BEM)

Reply:
Okay not BEM, but what is the convention you want to use? I think we need something to make it easy to relate the scss to the elements, or at least do something like `@wordpress/components` so the CSS is located close to the component files?

Reply:
https://github.com/WordPress/gutenberg/tree/73e6403789b278b48dafadbdc0bba737baae7afc/packages/components/src/button

Reply:
I don't like the idea of namespacing under an ID. otherwise we have to always use that selector when querying for styles within the source with JS.

Reply:
So let's table this discussion. It will take more thinking.